### DISCUSSION TITLE
Saving e-mail address separately

### DISCUSSION THREAD

**Discussion by lhunim95** [jspsych/jsPsych#2216] [Q&A] [answered]
Dear all,

the participants of my study have the possibility to participate in a raffle of vouchers. Therefore, I have to collect their e-mail address which should be saved separately from their data to ensure that the participants cannot be identified. 

I am running the experiment on cognition.run - so if someone has an idea how this might work, I'd be very thankful. :)

Best regards, 
Larissa

**Comment by becky-gilbert** [comment, marked as answer]
Hi Larissa,
Maybe you could do this by redirecting participants from one cognition.run study to another, where one is the actual study and the other just collects the email address? To make sure that it's not possible to link the data from the two studies together, you should not store IP addresses in at least one of the two studies. It might still be possible to link the data via timestamps. I'm not sure if it's possible to remove timestamps from the study results before they're stored by cognition.run, but if not then maybe it'd be enough to delete them from the downloaded data, and then delete the data from cognition.run.
I hope this helps!

**Reply by lhunim95** [reply]
thank you very much! Is there a function that automatically directs the participants to the other study?

**Reply by becky-gilbert** [reply]
Hi @lhunim95, this discussion post shows how to redirect participants to a different URL at the end of an experiment: https://github.com/jspsych/jsPsych/discussions/1850#discussioncomment-1142537 Please let us know if you have any problems with this solution.
